lib/cn/csched_sp3_work.h

```diff
@@ -15,6 +15,7 @@
 
 /* clang-format off */
```

Comment:
Why is this here?

Reply:
I moved it up a few lines, it should be format-on...

Reply:
Sorry, it's there because stupid clang format, which is not ready for prime time, kept effing up the spacing in the #defines making them difficult to read.  So same reason you see that anywhere in our code...

Reply:
Right, I am thinking just make it as small a clang-format ignore portion as possible. I didn't see why it is turned off prior to the the struct definitions.

Reply:
Hmm, I think it covers the structs because clang-format wrecks the spacing on the comments?